root/statistics/Editors.js

```diff
@@ -0,0 +1,128 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {l_statistics as l} from '../static/scripts/common/i18n/statistics';
+import EditorLink from '../static/scripts/common/components/EditorLink';
+import {withCatalystContext} from '../context';
+import loopParity from '../utility/loopParity';
+
+import {formatCount} from './utilities';
+import StatisticsLayout from './StatisticsLayout';
+
+type EditorsStatsT = {|
+  +dateCollected: string,
+  +topEditors: $ReadOnlyArray<EditorStatT>,
+  +topRecentlyActiveEditors: $ReadOnlyArray<EditorStatT>,
+  +topRecentlyActiveVoters: $ReadOnlyArray<EditorStatT>,
+  +topVoters: $ReadOnlyArray<EditorStatT>,
+|};
+
+type EditorStatT = {|
+  +count: number,
+  +entity: EditorT,
+|};
+
+const EditorStatsTable = withCatalystContext(({
+  $c,
+  countLabel,
+  dataPoints,
+  editorLabel,
+  tableLabel,
+}) => (
+  <>
+    <h3>{tableLabel}</h3>
+    <table className="tbl">
+      <thead>
+        <tr>
+          <th className="pos">{l('Rank')}</th>
+          <th>
+            {editorLabel}
+            <div className="arrow" />
+          </th>
+          <th>
+            {countLabel}
+            <div className="arrow" />
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        {dataPoints.length > 0 ? (
+          dataPoints.map((editorStat, index) => (
+            <tr className={loopParity(index)} key={editorStat.editor.id}>
```

Comment:
Am I missing something? `EditorStatT` has an `entity` prop, but here and below you're referencing an `editor` prop.

Reply:
Also, ``key={editorStat.editor ? editorStat.editor.id : `missing-${index}`}`` (or s/editor/entity/?) seems like it'd resolve @yvanzo's error.

Reply:
Duh. EditorStatsTable was missing types. It works, so I assume the EditorStatT type should have editor not entity. Will check.

Reply:
Thanks, page is not crashing anymore for me!